akka-docs/src/main/paradox/typed/mailboxes.md

```diff
@@ -63,43 +63,43 @@ fall-back to the default mailbox configuration section.
 Akka ships with a number of mailbox implementations:
 
  * 
-   **SingleConsumerOnlyUnboundedMailbox** (default)
+   @apidoc[SingleConsumerOnlyUnboundedMailbox](SingleConsumerOnlyUnboundedMailbox) (default)
```

Comment:
I should use the class name `SingleComsumerOnlyUnboundedMailBox`, because of `(default)`. Without class name I get error `No matches found for apidoc query [default] `

Reply:
I think that's fine